content/api/embeds/embeds_create.md

```diff
@@ -14,23 +14,24 @@ Returns: A JSON consisting of the same elements returned by GET api/v1/graph/emb
 Note: If an embed already exists for the exact same query in a given organization, the older embed is returned instead of creating a new embed.
 
 ##### ARGUMENTS
-* **`graph_json`** [*required*]:  
+
+*   **`graph_json`** [*required*]:
```

Comment:
This file has new indentations that aren't part of the monitor lists PR. Is this necessary?